src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -26,16 +27,35 @@ Future<Integer> loadData(TenantAttributes attributes, String tenantId,
           TenantLoading tl = new TenantLoading();
 
           tl.withKey("loadReference").withLead("ref-data");
-          tl.withIdContent();
-          tl.add("groups");
-          tl.withIdContent();
-          tl.add("addresstypes");
+          if (isNew(attributes, "15.4.0")) {
+            tl.withIdContent().add("addresstypes-15.4.0", "addresstypes");
+          }
+          if (isNew(attributes, "17.3.0")) {
+            tl.withIdContent().add("groups-17.3.0", "groups");
+          }
 
           tl.withKey("loadSample").withLead("sample-data");
-          tl.withIdContent();
-          tl.add("users");
+          if (isNew(attributes, "15.4.0")) {
+            tl.withIdContent().add("users-15.4.0", "users");
+          }
+          if (isNew(attributes, "17.3.0")) {
+            tl.withIdContent().add("users-17.3.0", "users");
+          }
```

Comment:
Can you provide any additional details about how the TenantRefAPI is used in upgrading? How do these changes reduce extra work for staff? Will this break upgrades?